lib/schema.js

```diff
@@ -324,7 +326,7 @@ function PrimitiveSchema(type) {
 
 util.inherits(PrimitiveSchema, Schema);
 
-function FieldSchema(name, type) {
+function FieldSchema(name, type, defaultValue) {
```

Comment:
@kyriesent Let me know when you have a few minutes to go through this together.  I can help you update the other code.